### PR TITLE
[Nav] Adds Github Starring to Nav

### PR DESCRIPTION
**Pull request by eliasjpr**
In order to gain more exposure and and promote the project this adds
github star button to the main navigation.

### CODE REVIEW

**Comment by faustinoaq** [comment]
Why doesn't show stars counter?

Like in footer

screenshot_20171204_114601

**Comment by eliasjpr** [comment]
@elorest added the count as per your suggestion

**Comment by faustinoaq** [comment]
There is one mistake here:

Chrome Console outputs:

```
> Uncaught SyntaxError: Unexpected identifier
<script>async defer src="https://buttons.github.io/buttons.js"</script>
```

File: src/views/layouts/application.slang [hunk 1]
```diff
@@ -10,6 +10,7 @@ html lang="en"
     link rel="apple-touch-icon" href="/favicon.png"
     link rel="icon" href="/favicon.png"
     link rel="icon" type="image/x-icon" href="/favicon.ico"
+    script async defer src="https://buttons.github.io/buttons.js"
```
Comment (faustinoaq) [inline src/views/layouts/application.slang]: It should be `script async="" defer="" src="https://buttons.github.io/buttons.js"`

**Comment by elorest** [comment]
Could you open that Faustino?

On Dec 4, 2017 10:15 AM, "Faustino Aguilar" <notifications@github.com>
wrote:

*@faustinoaq* commented on this pull request.

Another PR is needed to fix this small issue:

> Uncaught SyntaxError: Unexpected identifier
<script>async defer src="https://buttons.github.io/buttons.js"</script>

------------------------------

In src/views/layouts/application.slang
<https://github.com/amberframework/amberframework.org/pull/75#discussion_r154713448>
:

> @@ -10,6 +10,7 @@ html lang="en"
     link rel="apple-touch-icon" href="/favicon.png"
     link rel="icon" href="/favicon.png"
     link rel="icon" type="image/x-icon" href="/favicon.ico"
+    script async defer src="https://buttons.github.io/buttons.js"

It should be script async="" defer="" src="https://buttons.github.
io/buttons.js"

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/amberframework/amberframework.org/pull/75#pullrequestreview-80917673>,
or mute the thread

File: src/views/layouts/application.slang (hunk 1 above)
Comment (faustinoaq) [reply src/views/layouts/application.slang]: Well, I just realize, we don't need this line at all because already exist at before body end

**Comment by faustinoaq** [comment]
@elorest Done! 👍 #76

**Pull request merged**